src/lib.rs

```diff
@@ -714,6 +714,12 @@ impl ConfigFile {
     self.json.imports.is_some() || self.json.scopes.is_some()
   }
```

Comment:
This method right here is called `is_an_import_map`... probably could drop the `an` eventually...